index.html

```diff
@@ -1,16 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel = stylesheet type= text/css href="style.css"/>
+  </head>
+  <body>
+    <header class="head">
+      <figure>
+        <img class="logo" src="images/sport-car.jpg" alt="car-logo" />
+        <figcaption class="contact-info">
+          <p>Contact us</p>
+        </figcaption>
+      </figure>
+      <nav class="nav-menu">
+        <ul class="list">
+          <li class="list-item">Home</li>
+          <li class="list-item">Collection</li>
+          <li class="list-item">About</li>
+          <li class="list-item">Service</li>
+          <li class="list-item">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-container">
+      <aside class="sidebar">
+        <h1 class="title">Hello</h1>
+        <p class="info">information</p>
+      </aside>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+      <section class="main-img-container">
+        <img class="main-image" src="images/vw beetle.jpg" alt="vw-beetle" />
+      </section>
+      <section class="carosel-container">
+        <span id="image-set1-1"></span>
+        <span id="image-set1-2"></span>
+        <span id="image-set1-3"></span>
+        <span id="image-set1-4"></span>
 
-</html>
\ No newline at end of file
+        <figure class="image-carosel">
+          <img
+            class="image-set1"
+            src="images/Aston-Martin-DB5-1965-Copyright-of-pyntofmyld-Flickr-4-e1480514383676.jpg" 
+            id ='image1'
+            alt="Aston martin"
+          />
+          <img
+            class="image-set1"
+            src="images/citroen_ds21.jpg" id='image2'
+            alt="citroen_ds21"
+          />
+          <img
+            class="image-set1"
+            src="images/ford mustang.jpg"
+            alt="ford mustang"
+          />
+          <img class="image-set1" src="images/MG gt.jpg" alt="MG gt" />
+        </figure>
+      </section>
+      <section class="button-container">
+        <a href="#image-set1-1" class="image-button"></a>
+        <a href="#image-set1-2" class="image-button" id='image2'></a>
+        <a href="#image-set1-3" class="image-button"></a>
+        <a href="#image-set1-4" class="image-button"></a>
+      </section>
+      <section class="section-container">
+        <ul class="image-tag2">
+          <li>
```

Comment:
If you're using a list and list items, use a list item for each image. 
At the moment you only have one item in your list.